updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0
+
 :: Update hosts file
-python updateHostsFile.py -a
+if exist updateHostsFile.exe (
+    echo run exe
+    updateHostsFile --auto --extensions fakenews social gambling
+) else (
+    echo run script
+    python updateHostsFile.py --auto --extensions fakenews social gambling
+)
 
 :: Move new hosts file in-place
 COPY hosts %WINDIR%\System32\drivers\etc\
 
 :: Flush the DNS cache
 ipconfig /flushdns
+
+:endofscript
+pause 5
```

Comment:
Why a pause?

Reply:
When running the batch file by clicking on it, as opposed to the command line, it closed right away and I wanted to confirm it was actually doing what I thought it was.

Reply:
Okay we can keep it!